src/app/app.component.ts

```diff
@@ -68,10 +70,16 @@ export class AppComponent implements OnInit {
       () => (this.loading = false),
       (error: HttpErrorResponse) => {
         this.loading = false;
-        this.toasterService.popAsync(
-          'error', 'Error while loading config from config server, using default values instead' + error.message).subscribe(
-          () => console.log('default values applied')
-        )
+        if(error.statusText === this.NOT_FOUND){
```

Comment:
can you improve the if condition by comparing the status code (404) instead of the status text?